docs/add-ins/get-and-set-recurrence.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Get and set recurrence in an Outlook add-in
+description: Get and set various recurrence properties of an item in an Outlook add-in.
+author: elizs
+ms.topic: article
+ms.technology: office-add-ins
+ms.date: 11/28/2018
+ms.author: elizs
+---
+
+# Get and set recurrence
+
+Imagine that you need to schedule a meeting with your team to discuss the current status of a project. It turns out that you need the status repeatedly until the project is completed. Instead of scheduling a meeting each time, you can configure the meeting to include how often you want it to occur. Then let's say that subsequently one of the meetings is at a time where you or one of the key attendees can't make it. You can then update that particular instance or occurrence, or cancel it altogether, without affecting the remainder of the series. Note that you can also set up a recurrence for your repeat appointments.
+
```

Comment:
@ElizabethSamuel-MSFT I'm fine with David's proposal in general, but would suggest you avoid using the term "meeting" -- since recurrence can apply to appointments (which have no attendees) as well as meetings (which do have attendees).